packages/terra-action-footer/package.json

```diff
@@ -39,11 +39,11 @@
     "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
     "lint:scss": "stylelint src/**/*.scss",
     "precompile": "rm -rf lib",
-    "test": "npm run jest && npm run wdio",
+    "test": "npm run lint && npm run jest && npm run wdio",
     "jest": "jest --config ../../jestconfig.js",
-    "wdio-default": "cd ../.. && npx wdio ./packages/$npm_package_name/wdio.conf.js",
-    "wdio-lowlight": "cd ../.. && THEME=clinical-lowlight-theme npx wdio ./packages/$npm_package_name/clinical-lowlight-wdio.conf.js",
-    "wdio-fusion": "cd ../.. && THEME=orion-fusion-theme npx wdio ./packages/$npm_package_name/wdio.conf.js",
+    "wdio-default": "cd ../.. && npx wdio",
+    "wdio-lowlight": "cd ../.. && THEME=clinical-lowlight-theme npx wdio",
+    "wdio-fusion": "cd ../.. && THEME=orion-fusion-theme npx wdio",
```

Comment:
Any interest in exposing bin commands from root or a similar concept to simplify the maintenance/uniformity of these packages?

tc = terra-core
```json
    "test": "tc-test",
    "jest": "tc-jest",
    "wdio-default": "tc-wdio-default",
    "wdio-lowlight": "tc-wdio-lowlight",
    "wdio-fusion": "tc-wdio-fusion",
```

Reply:
I think this would go into the bucket of scripts we could add if we created a terra-scripts package. running wdio, jest, the changelog update script, etc. are scripts we've been duplicating across terra-core, framework and clinical.

If you have a strong opinion to do it now, I'd be more than happy to make the change.

Reply:
If the scripts package contains scripts like this we should be clear that it's for mono-repo/internal use only and not intended for end users.

Reply:
100% agreed. It'd be for ourselves.

Reply:
I don't have a strong opinion about making this change. There are advantages and disadvantages in obscuring the testing scripts. If we're interested in the idea we can talk more about the terra-script package in more detail down the road. 

We don't make changes to the scripts/commands too often, but when we do we make updates to all packages, it seems like a hassle to get right/write a script to make the updates to each package individually.

A little wordy, but the mono repo specific scripts could be namespaced. (If we wanted commands for non-mono and mono)

```
tc-mono-wdio-default

tcm-wdio-default
```

Reply:
I don't think it's a bad idea. Let's keep it in mind and we can do that work on a new PR. I'll merge this for now & we can solidify what we thinking might be best. I agree, it's a little annoying to update 15+ package.json for a simple script